package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.6",
   "description": "Send metrics from the browser to Librato",
   "main": "lib/librato-client.js",
+  "files": [ "src" ],
```

Comment:
@luk3thomas We need this in order for the package include to work in the spaces side. 
Let me know if you want to merge this PR or you prefer to add it in a separate PR.